script/build/image

```diff
@@ -1,11 +1,9 @@
-#!/bin/bash
+#!/bin/sh -e
```

Comment:
Use `/usr/bin/env sh` instead. On macOS and many Linux distros, `/bin/sh` is bash. By using `env`, you give the user the opportunity to use a POSIX shell like dash by respecting their configured `$PATH`.

It also makes my life easier when I'm testing a script I want to be POSIX-compliant. :)

Reply:
Thank you for your input, the patch has been updated.